src/scripts/init.js

```diff
@@ -0,0 +1,36 @@
+const appName = require('../../package.json').name;
+
+$(document).ready(function () {
+
+	const mountPoint = $('<div>', {
+		id: appName,
+		html: '<div>'
+	});
+
+	if (!OCA.Files) {
+		return;
+	}
+
+	// ---- Register fileactions -------
+
+	let ViewMedia = {
+		mime: 'image/png',
+		name: appName,
+		permissions: OC.PERMISSION_READ,
+		actionHandler(fileName, context) {
+
+			window[appName] = {
+				fileName,
+				context
+			};
+
+			window.location = ['#', appName, fileName].join('/');
```

Comment:
`window.hash` as discussed, or did it not work out ?

Reply:
Did not work for some reason … apparently it only reads.